esti/import_test.go

```diff
@@ -78,7 +122,9 @@ func verifyImportObjects(t *testing.T, ctx context.Context, repoName string, pre
 		})
 		require.NoError(t, err, "get object failed: %s", objPath)
 		require.Equal(t, http.StatusOK, objResp.StatusCode(), "get object %s", objPath)
-		require.Equal(t, expectedContentLength, int(objResp.HTTPResponse.ContentLength), "object content length %s", objPath)
+		if expectedContentLength > 0 {
```

Comment:
```suggestion
		if expectedContentLength >= 0 {
```

Reply:
Wanted to skip the size check in case of local block adapter.
I can switch check that the content contains the physical address or match the body size or the entry size.
What do you think?